### PR TITLE
Add initial overview

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

**Comment by sbc100** [comment]
Are we supposed to get reviews for this kind of thing before landing?

**Review by tlively — APPROVED** [review]
It's not a bad idea to get reviews from stakeholders on design changes, but for editorial things and initially setting up the proposal repo, you should feel free to make changes as champion without waiting on any reviews.

File: proposals/extended-const/Overview.md
Comment (rossberg) [inline proposals/extended-const/Overview.md:44]: I think you'll want at least multiplication as well, e.g., when an offset is a multiple of some count.

Comment (sbc100) [reply proposals/extended-const/Overview.md:44]: Sounds good, add add those too.

We can also add more (div?,  float?) further down the road of course.

**Pull request merged**
